cmd/fleetctl/get.go

```diff
@@ -1125,8 +1125,12 @@ func getMDMAppleBMCommand() *cli.Command {
 				}
 				return fmt.Errorf("could not get Apple BM information: %w", err)
 			}
+			config, err := client.GetAppConfig()
+			if err != nil {
+				return fmt.Errorf("could not get app config: %w", err)
+			}
 
-			defaultTeam := bm.DefaultTeam
+			defaultTeam := config.MDM.AppleBMDefaultTeam
```

Comment:
Ah, sorry there's been a misunderstanding here - the default team should be provided as part of the API call that is done by `client.GetAppleBM()` above (line 1119), so there wouldn't be any change needed here. The default team is part of the response payload returned by `GET /api/v1/fleet/mdm/apple_bm`.

Reply:
This is where it should be set: https://github.com/fleetdm/fleet/blob/main/ee/server/service/mdm.go#L43-L45